src/ncp/ncp_base_radio.cpp

```diff
@@ -123,11 +129,17 @@ void NcpBase::LinkRawTransmitDone(otRadioFrame *aFrame, otRadioFrame *aAckFrame,
             SuccessOrExit(mEncoder.WriteInt8(-128));                           // Noise Floor (Currently unused)
             SuccessOrExit(mEncoder.WriteUint16(0));                            // Flags
 
-            SuccessOrExit(mEncoder.OpenStruct()); // PHY-data
+            SuccessOrExit(mEncoder.OpenStruct());                                // PHY-data
+            SuccessOrExit(mEncoder.WriteUint8(aAckFrame->mChannel));             // Receive channel
+            SuccessOrExit(mEncoder.WriteUint8(aAckFrame->mInfo.mRxInfo.mLqi));   // Link Quality Indicator
+            SuccessOrExit(mEncoder.WriteUint32(aAckFrame->mInfo.mRxInfo.mMsec)); // The timestamp milliseconds
+            SuccessOrExit(
+                mEncoder.WriteUint16(aAckFrame->mInfo.mRxInfo.mUsec)); // The timestamp microseconds, offset to mMsec
 
-            SuccessOrExit(mEncoder.WriteUint8(aAckFrame->mChannel));           // Receive channel
-            SuccessOrExit(mEncoder.WriteUint8(aAckFrame->mInfo.mRxInfo.mLqi)); // Link Quality Indicator
+            SuccessOrExit(mEncoder.CloseStruct());
 
+            SuccessOrExit(mEncoder.OpenStruct());            // Vendor-data
+            SuccessOrExit(mEncoder.WriteUintPacked(aError)); // Receive error
```

Comment:
Should be translated to a `spinel_status_t` enumeration value? Otherwise the spinel protocol has a dependency on an external enumeration

Reply:
This field is in vendor specific data structure. And it's using the `UintPacked` data type, which doesn't depend on the enumeration.